### DISCUSSION TITLE
Writing chant notation in MuseScore 4?

### DISCUSSION THREAD

**Discussion by lbssousa** [musescore/MuseScore#18602] [Q&A]
Is there any updated documentation for writing chant notation (in particular, gregorian chant transcription) in MuseScore 4?

**Comment by MarcSabatella** [comment]
I don't think there have been any significant changes to how this works - the same techniques that worked in MU3 should still work, other than things like Inspector now being Properties etc.  Was there specific documentation you were using previously?  The Handbook section on mensural notation contains some info - https://musescore.org/en/handbook/4/mensural-notation-and-mensurstrich - and my online course [Mastering MuseScore 4](https://school.masteringmusescore.com/p/musescore4) quite a bit more.
